packages/block-editor/src/components/block-list/block.js

```diff
@@ -248,7 +248,10 @@ export class BlockListBlock extends Component {
 	 * @return {void}
 	 */
 	preventDrag( event ) {
-		event.preventDefault();
+		const { type } = JSON.parse( event.dataTransfer.getData( 'text' ) );
+		if ( type !== 'image' ) {
```

Comment:
Does this create an exception to `preventDrag` for all image elements within blocks, regardless if they're meant for dragging or not?

Would there be some more generic way to also allow blocks to implement drag-and-dropping of non-image elements?